### PR TITLE
Issue / 8725 Extract Subtle Notification Styling

### PR DESCRIPTION
**Pull request by 10upsimon**
## Summary

Addresses issue:

- #8725

## Relevant technical choices

- Created a new `SubtleNotification` component to centralise implementation for subtle notification banners
- Updated various banners making use if said styling to instead incorporate said new component
- Updated and added VRT tests accordingly

## PR Author Checklist

- [x] My code is tested and passes existing unit tests.
- [x] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 5.2 and PHP 7.4.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Code Reviewer Checklist

- [ ] Run the code.
- [ ] Ensure the acceptance criteria are satisfied.
- [ ] Reassess the implementation with the IB.
- [ ] Ensure no unrelated changes are included.
- [ ] Ensure CI checks pass.
- [ ] Check Storybook where applicable.
- [ ] Ensure there is a QA Brief.

## Merge Reviewer Checklist

- [ ] Ensure the PR has the correct target branch.
- [ ] Double-check that the PR is okay to be merged.
- [ ] Ensure the corresponding issue has a ZenHub release assigned.
- [ ] Add a changelog message to the issue.

### CODE REVIEW

File: assets/js/modules/ads/components/notifications/PAXSetupSuccessSubtleNotification.js [hunk 1]
```diff
@@ -61,44 +60,25 @@ export default function PAXSetupSuccessSubtleNotification() {
...
+			description={ __(
+				'Track your conversions, measure your campaign results and make the most of your ad spend',
+				'google-site-kit'
+			) }
+			onDismiss={ onDismiss }
+			additionalCTA={
+				<Button
+					onClick={ ( event ) => {
+						scrollToWidget( event );
+					} }
```
Comment (eugene-manuilov) [inline assets/js/modules/ads/components/notifications/PAXSetupSuccessSubtleNotification.js:77]:
```suggestion
					onClick={ scrollToWidget }
```

File: assets/js/modules/analytics-4/components/SubtleNotification.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { Button } from 'googlesitekit-components';
+import CheckFill from '../../../../svg/icons/check-fill.svg';
+import { Grid, Cell, Row } from '../../../material-components';
+import PropTypes from 'prop-types';
```
Comment (eugene-manuilov) [inline assets/js/modules/analytics-4/components/SubtleNotification.js:30]: This is an external dependency.

**Review by eugene-manuilov — CHANGES_REQUESTED** [review]
Thanks, @10upsimon. Looks good to me but there are two minor issues that we need to address.

File: assets/js/modules/analytics-4/components/SubtleNotification.js (hunk 2 above)
Comment (10upsimon) [reply assets/js/modules/analytics-4/components/SubtleNotification.js]: Thanks @eugene-manuilov I moved this dependency to an external section in my latest commit.

File: assets/js/modules/ads/components/notifications/PAXSetupSuccessSubtleNotification.js (hunk 1 above)
Comment (10upsimon) [reply assets/js/modules/ads/components/notifications/PAXSetupSuccessSubtleNotification.js]: Thanks @eugene-manuilov sorted in latest commit.

File: assets/js/modules/analytics-4/components/SubtleNotification.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
```
Comment (eugene-manuilov) [inline assets/js/modules/analytics-4/components/SubtleNotification.js:27]: @10upsimon, please, put the external dependencies section to be before WP deps.

**Review by eugene-manuilov — CHANGES_REQUESTED** [review]

File: assets/js/modules/analytics-4/components/SubtleNotification.js (hunk 3 above)
Comment (eugene-manuilov) [reply assets/js/modules/analytics-4/components/SubtleNotification.js:27]: Actually, i'll create a suggestion and apply it myself.

Comment (eugene-manuilov) [inline assets/js/modules/analytics-4/components/SubtleNotification.js:27]:
```suggestion
/**
 * External dependencies
 */
import PropTypes from 'prop-types';

/**
 * WordPress dependencies
 */
import { __ } from '@wordpress/i18n';
```

**Review by eugene-manuilov — CHANGES_REQUESTED** [review]

**Review by eugene-manuilov — APPROVED** [review]

**Pull request merged**